presto-main/src/main/java/com/facebook/presto/sql/planner/LogicalPlanner.java

```diff
@@ -69,6 +69,11 @@
 
 public class LogicalPlanner
 {
+    public enum Stage
```

Comment:
What's this for? It doesn't seem to have any effect, since the only value that's ever used is OPTIMIZED_AND_VALIDATED.